code/modules/atmospherics/gasmixtures/gas_mixture.dm

```diff
@@ -8,6 +8,8 @@ What are the archived variables for?
 
 /datum/gas_mixture
 	/// Never ever set this variable, hooked into vv_get_var for view variables viewing.
+	var/list/gases = list()
+	var/temperature = 0 //kelvins
```

Comment:
ditto, remove this

Reply:
Yep, removed it will commit in a second.

Reply:
Should be removed.